lib/challenge.js

```diff
@@ -231,11 +264,11 @@ let nums = [
 function prompt11() {
   let median;
   // YOUR CODE HERE
-
+  median = nums[Math.floor(nums.length/ 2)];
   console.log(median);
 }
 // COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
-// prompt11();
+// prompt11(); 
 
 // 🌟 MAKE A COMMIT: "Complete prompt 11"
 
```

Comment:
Watch the spacing after `nums.length` to make it more readable I would write it `median = nums[Math.floor(nums.length / 2)];` with a space around the `/` operator

This is a good start.  Challenge yourself to complete it. (no resubmit required)

> * Find the median number in the following nums array (the mid point), then
>  * console.log that number. The median number is the middle number in a SORTED list of numbers. Think through the steps you'll need to take to determine this number with code!